server/db/populate-db.sql

```diff
@@ -4,38 +4,42 @@ INSERT INTO users (email, password) values ('admin@cyf.org', 'admin_password');
 INSERT INTO users (email, password) values ('user@cyf.org', 'user_password');
 
 
- 
- INSERT INTO topics ( topic_name) values ('english');
- INSERT INTO topics ( topic_name) values ('get ready for work');
+INSERT INTO blocks (block_id, block_name) VALUES(1, 'Block 1');
+INSERT INTO blocks (block_id, block_name) VALUES(2, 'Block 2');
+
+
+INSERT INTO topics (topic_id, topic_name, document_name, document_link) VALUES(1, 'Beginners English', 'Beginners English', 'www.google.com');
+INSERT INTO topics (topic_id, topic_name, document_name, document_link) VALUES(2, 'Intermediate English', 'Intermediate English', 'www.bing.com');
+
+
+
+INSERT INTO block_topics (block_id, topic_id) VALUES(1, 1);
+INSERT INTO block_topics (block_id, topic_id) VALUES(1, 2);
+INSERT INTO block_topics (block_id, topic_id) VALUES(2, 1);
+INSERT INTO block_topics (block_id, topic_id) VALUES(2, 2);
+
+
+INSERT INTO volunteer_topic (volunteer_topic_id, topic_id, volunteer_id) VALUES(1, 1, 1);
```

Comment:
volunteer_topic_id is set to GENERATED so you cannot insert a value into it here.

You are trying to insert a volunteer_id before the volunteer table is populated, you need to move the section 33-34 to above this section.